### PR TITLE
Update 大话西游.md

### PR DESCRIPTION
**Pull request by laurie99**
added a line with a link @casey-collab

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

File: Chinese/大话西游.md [hunk 1]
```diff
@@ -30,4 +30,6 @@
 
 [灰公主](Cinderella/Cinderella.md)
 
-[校园偶像](school-idols/school-idols.md)?
\ No newline at end of file
+[校园偶像](school-idols/school-idols.md)?
+
+[福尔摩斯](sherlockholmes/sherlockholmes.md)?
```
Comment (cbuckey-uda) [inline Chinese/大话西游.md]: This link isn't working. Did you forget to commit the `sherlockholmes.md` file?

**Comment by cbuckey-uda** [comment]
It looks like there is a conflict. Could you merge the master branch of the original repo into your branch, resolve the conflicts, and update the pull request?

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**